### PR TITLE
Test for all cryptids

### LINKED ISSUE #25: BE - Add tests for cryptids query
Test for all cryptids

### PR DESCRIPTION
**Pull request by dkulback**
## Neccesary checkmarks:

    [x] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [x] Refactor 
 

## Description of Change:

    description:     changed query cryptids => get_cryptids + added test 

## Requested feedback:

    I'd like feedback on...

## Check the correct boxes

    [x] This broke nothing
    [x] This broke some stuff
    [x] This broke everything

## Testing Changes

    [] No Tests have been changed
    [x] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

### CODE REVIEW

File: spec/graphql/queries/cryptids/get_cryptids_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
```
Comment (echon006) [inline spec/graphql/queries/cryptids/get_cryptids_spec.rb:2]: This test is looks amazing!

File: spec/graphql/queries/cryptids/get_cryptids_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            'range' => cryptid.range,
+            'image' => cryptid.image
+          }
+        end
+      )
+    end
+  end
+end
+
+def query
```
Comment (echon006) [inline spec/graphql/queries/cryptids/get_cryptids_spec.rb:29]: what is this part for? is it to mimic a request?

**Review by echon006 — APPROVED** [review]

File: spec/factories/cryptids.rb [hunk 3]
```diff
@@ -2,7 +2,8 @@
   factory :cryptid do
     name { Faker::TvShows::Supernatural.creature }
     range { Faker::Address.state }
-    danger_level { Faker::Number.betwee(from: 1, to: 10) }
-    image { Faker::LoremFlickr.image(seach_terms: ['cryptid']) }
+    danger_level { Faker::Number.between(from: 1, to: 10) }
```
Comment (ParkerLockhart) [inline spec/factories/cryptids.rb:5]: So glad you caught this typo I made yesterday, I saw it this morning 😅

**Review by ParkerLockhart — APPROVED** [review]

File: spec/factories/cryptids.rb (hunk 3 above)
Comment (dkulback) [reply spec/factories/cryptids.rb:5]: no biggie quick fix thanks for setting it up with Faker!

File: spec/graphql/queries/cryptids/get_cryptids_spec.rb (hunk 2 above)
Comment (dkulback) [reply spec/graphql/queries/cryptids/get_cryptids_spec.rb:29]: OkCryptidBeSchema.execute(query).as_json - this line calls the query 

The test is checking to make sure it matches what the query requested. I took this from the legacy code review

https://github.com/DanielEFrampton/paired-be

So I'm open to new ideas on how we want to go about testing

**Pull request merged**